web/src/echo.ts

```diff
@@ -292,9 +392,9 @@ export function edit_locally(message, request) {
             // is important in case
             // markdown.contains_backend_only_syntax(message) is true.
             message.content = request.content;
-            message.mentioned = request.mentioned;
-            message.mentioned_me_directly = request.mentioned_me_directly;
-            message.alerted = request.alerted;
+            message.mentioned = request.mentioned ?? false;
+            message.mentioned_me_directly = request.mentioned_me_directly ?? false;
+            message.alerted = request.alerted ?? false;
```

Comment:
Hmm, are these not always available in these requests?

Reply:
Nope.